ui/DesignSystem/Primitive/Icon/Spinner.svelte

```diff
@@ -19,7 +17,15 @@
   }
 </style>
 
-<IconWrapper on:click {dataCy} {style}>
+<svg
```

Comment:
Perhaps adding a comment here would make sense so we don't refactor this away by accident sometime in the future.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/747/commits/51c08645f61459a2d8d9dd77c2c6b377ee1d9296